pkg/utils/utils.go

```diff
@@ -82,6 +83,17 @@ func DiscoverSriovDevices() ([]sriovnetworkv1.InterfaceExt, error) {
 			iface.Name = name
 		}
 
+		if iface.Name != "" {
+			if devLink, err := netlink.LinkByName(iface.Name); err == nil {
```

Comment:
devlink ?  maybe just `link` ? as to not confuse with linux devlink subsystem.
netlink.LinkByName retrieves rtnl link device